src/app.d.ts

```diff
@@ -0,0 +1,13 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+declare global {
+	namespace App {
```

Comment:
Whats this?

Reply:
App.d.ts is for getting the types to be safe at a local, global or platform level without having to import them every time you need them. Generated automatically in every web framework project using typescript.